yjit/src/core.rs

```diff
@@ -450,8 +450,11 @@ pub struct Context {
     /// Bitmap of which stack temps are in a register
     reg_temps: RegTemps,
 
-    // Depth of this block in the sidechain (eg: inline-cache chain)
-    chain_depth: u8,
+    /// Fields packed into u8
+    /// - Lower 7 bits: Depth of this block in the sidechain (eg: inline-cache chain)
+    /// - Top bit: Whether this code is the target of a JIT-to-JIT Ruby return
+    ///   ([Self::is_return_landing])
+    chain_depth_plus: u8,
```

Comment:
I'm not sure what `plus` means here. Is it "chain_depth plus (is_return_landing)"?

I would probably call it `chain_depth_return_landing` or `chain_depth_and_return_landing`, but they are too long I guess.

Reply:
![image](https://github.com/ruby/ruby/assets/713766/7064ba7f-da8c-47a9-b90d-f72f5dcb7f1d)